site/docs/getting-started/connect-providers/aws-bedrock.md

```diff
@@ -8,6 +8,12 @@ sidebar_position: 3
 
 This guide will help you configure Envoy AI Gateway to work with AWS Bedrock's foundation models.
 
+## Supported Features
```

Comment:
let's do this "support matrix" in another PR for #705, so can you revert this for now.